crates/nu-command/src/conversions/into/datetime.rs

```diff
@@ -371,7 +349,12 @@ mod tests {
     #[test]
     fn takes_iso8601_date_format() {
         let date_str = Value::test_string("2020-08-04T16:39:18+00:00");
-        let actual = action(&date_str, &None, &None, Span::test_data());
+        let args = Arguments {
+            zone_options: None,
+            format_options: None,
+            cell_paths: None,
+        };
+        let actual = action(&date_str, &args, Span::test_data());
```

Comment:
A bit unfortunate that the tests have to take the extra structs (so test refactoring with its associated risks is required) but it is much more readable like this, which is nice!

Reply:
Yeah, I've thought about this, because we need to create an extra struct, so the lines of testing code is increased actually, but I think that's ok :)

Reply:
Looking at a diff in tests is always a bit hard but I think here the benefit of having tests new contributors can quickly read (even though its more lines) outweighs that cost.